libs/application/templates/complaints-to-althingi-ombudsman/src/lib/messages/attachments.ts

```diff
@@ -16,7 +16,7 @@ export const attachments = defineMessages({
   uploadDescription: {
     id: 'ctao.application:section.attachments.uploadDescription',
     defaultMessage:
-      'Tekið er við skjölum með endingunum: .pdf, .doc, .docx, .rtf, .jpg, .jpeg, .png, .heic',
+      'Tekið er við skjölum með endingunum: .pdf, .doc, .docx, .rtf, .jpg, .jpeg, .png, .heic, .xlsx, .xls',
```

Comment:
Maybe we should use the UPLOAD_ACCEPT constant here

Reply:
extract-strings does not like that unfortunately 😞